src/actuator/ActuatorBulkDiskFAST.C

```diff
@@ -48,6 +48,17 @@ ActuatorBulkDiskFAST::ActuatorBulkDiskFAST(
   initialize_swept_points(actMeta);
 }
 
+void
+ActuatorBulkDiskFAST::initialize_ADM_points(const ActuatorMetaFAST& actMeta)
+{
+  // LCC NOTE: Currently this is called every iteration -- incurring a
+  // noticeable performance penalty. Need to fix so that it's only
+  // called in the beginning (for fixed yaw), or called when the
+  // turbine yaw changes.
+  initialize_swept_points(actMeta);    
```

Comment:
```suggestion
if(!initialized_ADM_points){
  RunActFastUpdatePoints(*this);
  initialize_swept_points(actMeta);
  stk_search_act_pnts(actMeta, stkBulk_);
  initialized_ADM_points = true;
 }    
```

Reply:
I was looking through the commit history and I'm wondering if you tried this when you were investigating this hanging?  You'd need to change the function signature and remove these added calls from `ActuatorExecutorFASTNgp.C`.

Reply:
Sorry for the super long delay in response.  Yes, I think I did actually try something similar to that last year, but the combination that I put in there was the one that didn't hang.

Lawrence